components/resources/brave_components_strings.grd

```diff
@@ -414,6 +414,7 @@
       <message name="IDS_BRAVE_REWARDS_LOCAL_ADS_SUBDIVISION_TARGETING_DISABLED" desc="">Disabled</message>
       <message name="IDS_BRAVE_REWARDS_LOCAL_QR_BOX_TEXT" desc="">QR Code for iOS Rewards users transferring BAT</message>
       <message name="IDS_BRAVE_REWARDS_LOCAL_QR_BOX_BUTTON" desc="">QR Code</message>
+      <message name="IDS_BRAVE_REWARDS_LOCAL_SERVER_DISCONNECT_FAILED" desc="">Error disconnecting [[wallet]] wallet from Brave Rewards</message>
```

Comment:
We're using `<ph` place holders now, so instead of `[[wallet]]` we can use something like:

`<ph name="PROVIDER">$1<ex>Uphold</ex>`